### DISCUSSION TITLE
DLL load failed while importing _ssl [4084] Failed to execute script pyi_rth_certifi

### DISCUSSION THREAD

**Discussion by hymancheng** [pyinstaller/pyinstaller#5985] [Help]
I copied the packaged folder to other computers, and there was an error at runtime. 
I have tried use `--hidden-import=_ssl` in pyinstaller cmd, but it didn't work.

pyinstaller build os information as:
python version 3.8.10
OS: window 10 Pro

**Comment by rokm** [comment]
Is the OS on target computer also Windows 10? Are you using anaconda python or python.org build?

>  I have tried use --hidden-import=_ssl in pyinstaller cmd, but it didn't work.

That's because this is not a missing module problem, but rather failure to load a python extension (DLL). 

This is usually caused by a missing DLL dependency. Try using [dependency walker](https://www.dependencywalker.com/) to open the `_ssl.pyd` file on the target computer and see what DLL it is missing.

**Reply by hymancheng** [reply]
The target computer also Windows 10. But I also hope that it can run normally on win7 system. 
I was using anaconda python.

**Reply by hymancheng** [reply]
I uninstall anaconda, and downgraded Python to 3.6. 
Then it work normal on both computer.
